pkg/network/node/ovscontroller.go

```diff
@@ -36,8 +36,8 @@ const (
 	Vxlan0 = "vxlan0"
 
 	// rule versioning; increment each time flow rules change
-	ruleVersion = 10
-
+	ruleVersion      = 11
+	egressIPPriority = 100
```

Comment:
put that somewhere not between `ruleVersion` and `ruleVersionTable`, which really ought to be together

Reply:
Removing it altogether. As you mentioned here: https://github.com/openshift/sdn/pull/299/files#r633656457 I shouldn't even need it. The only reason I thought I did was because of what you mentioned here: https://github.com/openshift/sdn/pull/299/files#r633654820